src/runners/runner.rs

```diff
@@ -131,12 +133,23 @@ impl Session {
             _ => unreachable!(),
         };
         let templ = self.create_bash_script(template)?;
+
+        // Create a temporary file to store the script and make it executable
+        let mut temp_script = NamedTempFile::new()?;
+        temp_script.write_all(templ.as_bytes())?;
+        let mut perms = temp_script.as_file().metadata()?.permissions();
+        perms.set_mode(perms.mode() | 0o111);
+        temp_script.as_file().set_permissions(perms)?;
+        
+        println!("Executing runner script {}", temp_script.path().display());
```

Comment:
Not sure if you wanna output this

Reply:
Something like this [8681cbc](https://github.com/0xricksanchez/AFL_Runner/pull/45/commits/8681cbc9f0f0112bf7c8e680a8bab25a5ef9501b) ?

Reply:
I mean at that point the path is invalid anyway since the program is terminated and the temp file is cleaned up :facepalm: